### PR TITLE
Fix issue with pipeline_version url param.

### PR DESCRIPTION
**Pull request by MarkAZhang**
Also update query-string package.

# Description

This fixes an issue where if you are looking at a report with pipeline_version=3.10, on refresh the pipeline_version becomes 3.1, because the value was being parsed as an integer.

To do this, we had to update the query-string package because the "parseNumbers" option wasn't added until a later version.

# Notes

Also updated the UrlQueryParser to not parse booleans and numbers by default. In the previous implementation, numbers and booleans were automatically parsed.

# Tests

* Verified that sample report url params still work.
* Verified that data discovery url params still work.

### CODE REVIEW

**Comment by MarkAZhang** [comment]
Held off on refactoring `helpers/url` to use `UrlQueryParser` (which would require refactoring the fillUrlParam code in PipelineSampleReport) because presumably the report page refactor would remove that code. If this isn't the case, happy to do the refactor.

File: app/assets/src/components/utils/UrlQueryParser.js [hunk 1]
```diff
@@ -10,7 +10,10 @@ class UrlQueryParser {
   }
 
   parse(query) {
-    let params = QueryString.parse(query);
+    let params = QueryString.parse(query, {
+      parseNumbers: false,
```
Comment (tfrcarvalho) [inline app/assets/src/components/utils/UrlQueryParser.js]: According to the github repo, these values are actually the default (https://github.com/sindresorhus/query-string#parsenumbers)... why did you need this?

Comment (tfrcarvalho) [reply app/assets/src/components/utils/UrlQueryParser.js]: Also, UrlQueryParser is not used below, right?

Comment (MarkAZhang) [reply app/assets/src/components/utils/UrlQueryParser.js]: Thanks for the catch regarding the defaults. Don't need to make a change in this file then.
